### DISCUSSION TITLE
Security

### DISCUSSION THREAD

**Discussion by Rmfjd1978** [h2oai/wave#1655] [Q&A]
I've created a container in linux with an image of the wave server and with an image of a wave app. My question is when a user acess de container how can i get the active directory user that is acessing?

**Comment by mturoci** [comment]
Hi @Rmfjd1978, unfortunately, Wave only supports OIDC auth which means there is no native way to retrieve the AD user. You will need to connect to AD yourself and build out the login page around it in Wave on your own (compared to what is given to you by Wave out of the box).
